framework/set/resources/airgap/createMainTF.go

```diff
@@ -54,7 +55,12 @@ func CreateMainTF(t *testing.T, terraformOptions *terraform.Options, keyPath str
 		return "", "", err
 	}
 
-	terraform.InitAndApply(t, terraformOptions)
+	_, err = terraform.InitAndApplyE(t, terraformOptions)
+	if err != nil {
```

Comment:
Could we actually pass `rancherConfig` in to the function, and say something like:

```
if err !=nil && rancherConfig.cleanup {
```

This way, when troubleshooting and cleanup is disabled, the resources wouldn't get cleaned up under that scenario? 🙂  - this suggestion would apply to all of the error handling for `terraform.InitAndApplyE()`, but im just going to leave one comment 😅